sample-extension/EdFi.Ods.Extensions.Sk/Artifacts/MsSql/Data/Ods/0010-SampleExtensionsData.sql

```diff
@@ -3,33 +3,11 @@
 -- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 -- See the LICENSE and NOTICES files in the project root for more information.
 
-DECLARE @grandBendHighSchoolId INT;
-DECLARE @localEducationAgencyId INT;
-DECLARE @student604822USI INT;
-DECLARE @student604823USI INT;
-DECLARE @student604822EntryDate date;
-DECLARE @student604823EntryDate date;
-
-SELECT @grandBendHighSchoolId = SchoolId , @localEducationAgencyId = localEducationAgencyId FROM edfi.School WHERE SchoolId = 255901001
-SELECT @student604822USI = StudentUSI FROM edfi.Student WHERE StudentUniqueId = '604822'
-SELECT @student604823USI = StudentUSI FROM edfi.Student WHERE StudentUniqueId = '604823'
-SELECT @student604822EntryDate = EntryDate FROM edfi.[StudentSchoolAssociation] WHERE StudentUSI = @student604822USI
-SELECT @student604823EntryDate = EntryDate FROM edfi.[StudentSchoolAssociation] WHERE StudentUSI = @student604823USI
-
-
---Sanity check to make sure some data exists, otherwise skip the script
-
-IF (@grandBendHighSchoolId IS NULL OR @student604822USI IS NULL OR @student604823USI IS NULL) RETURN
-
-INSERT INTO [sk].[StudentSchoolAssociationExtension]
-([EntryDate],[SchoolId],[StudentUSI],[ResidentLocalEducationAgencyId],[ResidentSchoolId],[ReportingSchoolId])
-VALUES
-(@student604822EntryDate,@grandBendHighSchoolId,@student604822USI,@localEducationAgencyId,@grandBendHighSchoolId,@grandBendHighSchoolId)
-
-
-INSERT INTO [sk].[StudentSchoolAssociationExtension]
-([EntryDate],[SchoolId],[StudentUSI],[ResidentLocalEducationAgencyId],[ResidentSchoolId],[ReportingSchoolId])
-VALUES
-(@student604823EntryDate,@grandBendHighSchoolId,@student604823USI,@localEducationAgencyId,@grandBendHighSchoolId,@grandBendHighSchoolId)
+INSERT INTO sk.StudentSchoolAssociationExtension (EntryDate, SchoolId, StudentUSI, ResidentLocalEducationAgencyId, ResidentSchoolId, ReportingSchoolId)
+SELECT ssa.EntryDate, ssa.EducationOrganizationId, st.StudentUSI, sc.LocalEducationAgencyId, sc.SchoolId, sc.SchoolId
+FROM edfi.Student st
+JOIN edfi.StudentSchoolAssociation ssa ON st.StudentUSI = ssa.StudentUSI
+JOIN edfi.School sc ON ssa.EducationOrganizationId = sc.SchoolId
```

Comment:
This is joining the EducationOrganizationId (sourced from the optional Graduation Plan) to the SchoolId, when there's no guarantee that we'll have a GraduationPlan.EducationOrganizationId, or that it has to be school level.  Is the intent here for this to be joining SchoolId to SchoolId?